templates/actionpolicy.erb

```diff
@@ -1,7 +1,7 @@
 policy default <%= @data['default'] %>
 <%
     lines = @data['lines'].collect do |line|
-      line.values_at(*%w{ action callerid actions facts classes }).join("\t")
+      line.values_at(*%w{ action callerid actions fact_filter classes }).join("\t")
```

Comment:
This is not necessary, because the hash still has the key facts:
https://github.com/puppet-community/puppet-mcollective/pull/161/files#diff-26b416a07573e48cd8576ce727abff8aL18

You break the policy file with that chenge